Class/CleanupVMs.ps1

```diff
@@ -5,66 +5,99 @@ param
     [string] $LabName,
 
     # Credential path
-    [Parameter(Mandatory=$false, HelpMessage="Path to file with Azure credentials")]
-    [string] $credentialPath = "$env:APPDATA\AzProfile.txt"
+    [Parameter(Mandatory=$false, HelpMessage="Path to file with Azure Profile")]
+    [string] $profilePath = "$env:APPDATA\AzProfile.txt"
 )
 
 # Stops at the first error instead of continuing and potentially messing up things
-$global:erroractionpreference = 1
+#$global:erroractionpreference = 1
+$global:VerbosePreference = $VerbosePreference
```

Comment:
It's so the helper class, ClassHelper.psm1, can see the value for Verbose from the main script. For whatever reason, even though I am loading the helper module into the script, it stills runs in a different scope and thus does not see whether the -Verbose parameter is set or not. Setting the "global" version resolved this.